ground/src/main/java/com/google/android/ground/model/submission/DateTaskData.kt

```diff
@@ -15,26 +15,17 @@
  */
 package com.google.android.ground.model.submission
 
-import java.text.DateFormat
-import java.text.SimpleDateFormat
-import java.util.Date
-import java.util.Locale
-import kotlinx.serialization.Contextual
 import kotlinx.serialization.Serializable
 
 /** A user-provided response to a date question task. */
 @Serializable
-data class DateTaskData(val date: @Contextual Date) : TaskData {
-  // TODO(#752): Use device localization preferences.
-  private val dateFormat: @Contextual DateFormat =
-    SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+data class DateTaskData(val time: Long) : TaskData {
```

Comment:
Can we rename this class to `DateTimeTaskData` since it is used for both tasks?

Reply:
Done